src/main/resources/assets/bloodmagic/lang/es_ES.lang

```diff
@@ -0,0 +1,164 @@
+item.BloodMagic.baseComponent.reagentBounce.name=Catalizador de Elasticidad
+item.BloodMagic.baseComponent.reagentFrost.name=Catalizador de Escarcha
+
+item.BloodMagic.baseComponent.reagentTeleposition.name=Catalizador de Teleposición
+item.BloodMagic.baseComponent.reagentTransposition.name=Catalizador de Transposición
+
+item.BloodMagic.baseComponent.ironSand.name=Arena de Hierro
+item.BloodMagic.baseComponent.goldSand.name=Arena de Oro
+item.BloodMagic.baseComponent.coalSand.name=Arena de Carbón
+item.BloodMagic.baseComponent.plantOil.name=Aceite Vegetal
+item.BloodMagic.baseComponent.sulfur.name=Sulfuro
+item.BloodMagic.baseComponent.saltpeter.name=Salitre
+item.BloodMagic.baseComponent.neurotoxin.name=Neuro Toxina
+item.BloodMagic.baseComponent.antiseptic.name=Antiseptico
+
+item.BloodMagic.cuttingFluid.basicCuttingFluid.name=Fluído de Corte Básico
+item.BloodMagic.cuttingFluid.explosive.name=Polvo Explosivo
+
+item.BloodMagic.baseComponent.mundaneLength.name=Catalizador de Duracíon Simple
+item.BloodMagic.baseComponent.mundanePower.name=Catalizador de Potencia Simple
+
+
+item.BloodMagic.demonCrystal.crystalDefault.name=Cristal de Voluntad Demonáaca
+item.BloodMagic.demonCrystal.crystalCorrosive.name=Cristal de Voluntad Corrosiva
+item.BloodMagic.demonCrystal.crystalDestructive.name=Cristal de Voluntad Destructiva
+item.BloodMagic.demonCrystal.crystalVengeful.name=Cristal de Voluntad Vengativa
+item.BloodMagic.demonCrystal.crystalSteadfast.name=Cristal de Voluntad Tenaz
+
+item.BloodMagic.monsterSoul.base.name=Voluntad Demoníaca
+item.BloodMagic.monsterSoul.corrosive.name=Voluntad Demoníaca Corrosiva
+item.BloodMagic.monsterSoul.destructive.name=Voluntad Demoníaca Destructiva
+item.BloodMagic.monsterSoul.vengeful.name=Voluntad Demoníaca Vengativa
+item.BloodMagic.monsterSoul.steadfast.name=Voluntad Demoníaca Tenaz
+
+item.BloodMagic.sigil.air.name=Signo de Aire
+item.BloodMagic.sigil.bloodLight.name=Signo de la Lámpara de Sangre
+item.BloodMagic.sigil.compression.name=Signo de Compresión
+item.BloodMagic.sigil.divination.name=Signo de Divinación
+item.BloodMagic.sigil.water.name=Signo de Agua
+item.BloodMagic.sigil.lava.name=Signo de Lava
+item.BloodMagic.sigil.void.name=Signo de Vacío
+item.BloodMagic.sigil.greenGrove.name=Signo de la Verde Arboleda
+item.BloodMagic.sigil.elementalAffinity.name=Signo de Afinidad Elemental
+item.BloodMagic.sigil.haste.name=Signo de Velocidad
+item.BloodMagic.sigil.suppression.name=Signo de Supresión
+item.BloodMagic.sigil.magnetism.name=Signo de Magnetismo
+item.BloodMagic.sigil.fastMiner.name=Signo del Minero Veloz
+item.BloodMagic.sigil.seer.name=Signo del Vidente
+item.BloodMagic.sigil.phantomBridge.name=Signo del Puente Fantasmal
+item.BloodMagic.sigil.whirlwind.name=Signo del Remolino
+item.BloodMagic.sigil.enderSeverance.name=Signo de División Ender
+item.BloodMagic.sigil.holding.name=Signo de Almacenamiento
+item.BloodMagic.sigil.holding.display=&r%s: &o&n%s
```

Comment:
The single characters prefixed with a % (only s in this case) are for string replacements. The ones prefixed by & are for Minecraft formatting codes for colors and such.